docs/Build/grestsvcs.json

```diff
@@ -569,7 +577,7 @@
       }
     },
     "/epoch_info_cache": {
```

Comment:
We can actually remove epoch_info_cache (I had added it before myself), I think we only need to add a control table with status